paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass_tester.cc

```diff
@@ -161,6 +163,36 @@ ProgramDesc BuildConvMultiRequantProgramDesc(bool use_mkldnn, float scale_out,
   return prog;
 }
 
+// a->Conv1->b
+// b->Dequant1(Scale1)->c
+// c->Concat
+ProgramDesc BuildConvDequantConcatProgramDesc(bool use_mkldnn, float scale_out,
+                                              float scale) {
+  ProgramDesc prog;
+  for (auto& v : variable_names) {
+    prog.MutableBlock(0)->Var(v);
+  }
+  SetOp(&prog, "conv2d", "Conv1", {"a"}, {"b"}, use_mkldnn, scale_out);
+  SetOp(&prog, "dequantize", "Dequant1", {"b"}, {"c"}, use_mkldnn, scale);
+  SetOp(&prog, "concat", "Concat1", {"c"}, {"d"}, use_mkldnn);
+  return prog;
+}
+
+// a->Conv1->b
+// b->Dequant1(Scale1)->c
+// b->Conv2->d
+ProgramDesc BuildConvDequantConvProgramDesc(bool use_mkldnn, float scale_out,
```

Comment:
Here, the `Dequant1`'s output (`c`) is the input to a single `Conv2` only. Probably `Conv1` taking `c` as input is missing here.

Reply:
The test checks if nothing changes if we have more than one output from Conv1. So Conv1 sholud have two outputs.

Reply:
You are right. Thanks!